protos/flyteidl/admin/project.proto

```diff
@@ -15,6 +15,18 @@ message Domain {
     string name  = 2;
 }
 
+// The status of the project is used to control its visibility in the UI and validity.
+enum ProjectState {
```

Comment:
to avoid the fun of the repeated usage of the word 'project' in the generated enum values, can we wrap the enum in a message like https://john-millikin.com/effective-grpc#enums (I know NamedEntity does not follow this convention, unfortunately)